src/ui/zcl_abapgit_services_git.clas.abap

```diff
@@ -307,8 +313,11 @@ CLASS zcl_abapgit_services_git IMPLEMENTATION.
 
         zcl_abapgit_objects=>delete( it_tadir  = lt_selected
                                      is_checks = ls_checks ).
-* update repo cache
-        lo_repo->refresh( ).
+
+        IF iv_refresh = abap_true.
```

Comment:
what does this do? is it part of refactoring the popup functionality?

`IF iv_refresh = abap_true.`

Reply:
No, it doesn't directly belong to this PR - will remove it. Later via iv_refresh it's possible to control wether to refresh the repo or not to be able to suppress a unneccessary repo refresh.